extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -113,36 +120,36 @@ const Track = memo(
 						// Prevent default behavior (scrolling one page down).
 						e.preventDefault();
 
-					// Select track.
-					selectTrack( index );
-				} }
-			>
-				<TrackIcon
-					className="jetpack-podcast-player__episode-status-icon"
-					isPlaying={ isPlaying }
-					isError={ isError }
-				/>
-				<span className="jetpack-podcast-player__episode-title">{ track.title }</span>
-				{ track.duration && (
-					<time className="jetpack-podcast-player__episode-duration">{ track.duration }</time>
-				) }
-			</a>
-			{ isActive && isError && <TrackError link={ track.link } /> }
-		</li>
-	);
-} );
-
-const Playlist = memo(
-	( {
-		tracks,
-		selectTrack,
-		currentTrack,
-		playerState,
-		colors,
-	} ) => {
-		return (
-			<ol className="jetpack-podcast-player__episodes">
-				{ tracks.map( ( track, index ) => {
+						// Select track.
+						selectTrack( index );
+					} }
+				>
+					<TrackIcon
+						className="jetpack-podcast-player__episode-status-icon"
+						isPlaying={ isPlaying }
+						isError={ isError }
+					/>
+					<span className="jetpack-podcast-player__episode-title">{ track.title }</span>
+					{ track.duration && (
+						<time className="jetpack-podcast-player__episode-duration" datetime={ track.duration }>
+							{ track.duration }
+						</time>
+					) }
+				</a>
+				{ isActive && isError && <TrackError link={ track.link } /> }
+			</li>
+		);
+	}
+);
```

Comment:
Finding it hard to trace what is your change and what is just code formatting that's escaped the linter due to your use of `no-verify`. Hopefully when you update these changes will disappear?

Reply:
This file was linting fine. I did a rebase and updated my packages, so maybe that's why so extra lines showed up?

The change I recall from this section is adding the `datetime` to `<time>`: `datetime={ track.duration }`